common/txmgr/inmemory_store.go

```diff
@@ -272,7 +332,47 @@ func (ms *inMemoryStore[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) FindT
 	[]txmgrtypes.TxAttempt[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE],
 	error,
 ) {
-	return nil, nil
+	return ms.persistentTxStore.FindTxAttemptsConfirmedMissingReceipt(ctx, chainID)
+
+	// TODO: THE SORTING OF GAS PRICE AND GAS TIP CAP IS NOT IMPLEMENTED CORRECTLY. NEED TO FIGURE OUT HOW TO GET GAS PRICE AND GAS TIP CAP FROM TxFee
+	// THIS WORK WILL NEED TO BE DONE IN A SEPARATE PR WHICH CHANGES THE FEE GENERIC
+	/*
+		if ms.chainID.String() != chainID.String() {
+			panic("invalid chain ID")
+		}
+
+		txFilter := func(tx *txmgrtypes.Tx[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) bool {
+			return tx.TxAttempts != nil && len(tx.TxAttempts) > 0
+		}
+		txAttemptFilter := func(attempt *txmgrtypes.TxAttempt[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) bool {
+			return true
+		}
+		states := []txmgrtypes.TxState{TxConfirmedMissingReceipt}
+		attempts := []txmgrtypes.TxAttempt[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]{}
+		ms.addressStatesLock.RLock()
+		defer ms.addressStatesLock.RUnlock()
+		for _, as := range ms.addressStates {
+			attempts = append(attempts, as.findTxAttempts(states, txFilter, txAttemptFilter)...)
+		}
+		// sort by tx_id ASC, gas_price DESC, gas_tip_cap DESC
+		// TODO: FIGURE OUT HOW TO GET GAS PRICE AND GAS TIP CAP FROM TxFee
+		slices.SortFunc(attempts, func(a, b txmgrtypes.TxAttempt[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]) int {
+			aSequence, bSequence := a.Tx.Sequence, b.Tx.Sequence
+			if aSequence == nil || bSequence == nil {
+				return 0
+			}
+
+			return cmp.Compare((*aSequence).Int64(), (*bSequence).Int64())
+		})
+
+		// deep copy the attempts
+		var eAttempts []txmgrtypes.TxAttempt[CHAIN_ID, ADDR, TX_HASH, BLOCK_HASH, SEQ, FEE]
+		for _, attempt := range attempts {
+			eAttempts = append(eAttempts, ms.deepCopyTxAttempt(attempt.Tx, attempt))
+		}
+
+		return eAttempts, nil
+	*/
```

Comment:
Is there a ticket for this, and if not can you create one and include a link in the comment?